src/views/About.vue

```diff
@@ -1,5 +1,60 @@
 <template>
   <div class="about">
-    <h1>This is under construction.</h1>
+    <SideBar class='sidebar'/>
+    <section class='about-content'>
+      <h2>Turing School of Software & Design Capstone Project</h2>
+      <p>The final project at Turing school is a full-stack venture to create an app
+        around an idea pitched by one of the students. Our goal was to develop a
+        crowdfunding dApp that made use of blockchain technology.
+      </p><p>
+        We had two weeks to
+        wireframe a front-end to interact with the blockchain and a back-end to supplement
+        it. We decided to use <b>Python</b> (+ ...other technologies) for our database and
```

Comment:
Maybe instead of a paragraph, we could have something like
Technologies: Vue, Python, whatever...
Project length: 2 weeks
just to make it easier to read